dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -28,7 +28,7 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 
-{% if not skip_chassisd and IS_MODULAR_CHASSIS == 1 %}
+{% if include_chassisd or (not skip_chassisd and IS_MODULAR_CHASSIS == 1) %}
```

Comment:
@rameshraghupathy should this be like to make it clear we are making this change fro smart switch?
if not `skip_chassisd` and (IS_MODULAR_CHASSIS == 1 or IS_SMART_SWITCH)`

Reply:
We can make it  "if IS_SMART_SWITCH or (IS_MODULAR_CHASSIS == 1 and not skip_chassisd)"  We want to have it all the time when it is smartswitch

Reply:
@prgeor